osu.Game/Database/RealmAccess.cs

```diff
@@ -99,8 +99,9 @@ public class RealmAccess : IDisposable
         /// 47   2025-01-21    Remove right mouse button binding for absolute scroll. Never use mouse buttons (or scroll) for global actions.
         /// 48   2025-03-19    Clear online status for all qualified beatmaps (some were stuck in a qualified state due to local caching issues).
         /// 49   2025-06-10    Reset the LegacyOnlineID to -1 for all scores that have it set to 0 (which is semantically the same) for consistency of handling with OnlineID.
+        /// 50   2025-07-11    Add UserTags to BeatmapMetadata.
         /// </summary>
-        private const int schema_version = 49;
+        private const int schema_version = 50;
```

Comment:
Notably, this will cause version incompatibility when deployed to tachyon, so not sure we want to deploy immediately.